packages/frontend/workspace/src/affine/crud.ts

```diff
@@ -85,6 +77,16 @@ export const CRUD: WorkspaceCRUD<WorkspaceFlavour.AFFINE_CLOUD> = {
       })
     );
 
+    const datasource = createAffineDataSource(
+      createWorkspace.id,
+      newBlockSuiteWorkspace.doc,
+      newBlockSuiteWorkspace.awarenessStore.awareness
+    );
+
+    const disconnect = datasource.onDocUpdate(() => {});
```

Comment:
why there need this and disconnect immediately ?

Reply:
this is not the real `connection` but the datasource and listeners behind, we need to drop the resources occupied